CHANGELOG.md

```diff
@@ -23,6 +24,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Fixed
 - start: correct invocation of prestart and poststart hooks (#200)
+- start: make tinirc configure the address selection policy (#204)
```

Comment:
@pizzamig or should this say: `tinirc: configure the address the selection policy`
(seen both styles in the changelog so far)

Reply:
definitively not an issue.
In the current implementation, the feature is limited to tinirc, maybe `tinirc:` would be more appropriate.
If we can also implement address selection policy for no-tinirc jails, we could use `start:` or `network:`